server-protocol.md

```diff
@@ -186,16 +186,26 @@ observers to use the hard-earned token for themselves.
 Wormhole codes look like `4-purple-sausages`, consisting of a number followed
 by some random words. This number is called a "Nameplate".
 
-On the Rendezvous Server, the Nameplate contains a pointer to a Mailbox.
-Clients can "claim" a nameplate, and then later "release" it. Each claim is
+On the Mailbox server, the Nameplate contains a pointer to a Mailbox.
+Clients can "claim" a nameplate, each claim is
 for a specific side (so one client claiming the same nameplate multiple times
-only counts as one claim). Nameplates are deleted once the last client has
-released it, or after some period of inactivity.
+only counts as one claim). There used to be an explicit `release` command, but
+Nameplates are new implicitly deleted once both clients have claimed
+it and thus found their mailbox, or after some period of inactivity. A client
+disconnecting with a `close` command also releases the nameplate. (A client
+disconnecting without a `close` command probably just has connection issues and
+is expected to reconnect after a short amount of time.)
 
 Clients can either make up nameplates themselves, or (more commonly) ask the
 server to allocate one for them. Allocating a nameplate automatically claims
-it (to avoid a race condition), but for simplicity, clients send a claim for
+it (to avoid a race condition), but for simplicity, clients can send a claim for
 all nameplates, even ones which they've allocated themselves.
+Note that a claim (and therefore an allocation) also automatically opens the
+mailbox, see below.
+The `claim` command has the optional boolean field `dont-allocate`, which if set
```

Comment:
Might be easier to read if this is flipped around (i.e. the boolean is `allocate` with implicit default of `true`).

Reply:
Thought of this too, the idea was that `false` is easier as default value. I like it either way around